src/middle-end/auth/index.js

```diff
@@ -63,19 +63,15 @@ module.exports = (m) => {
                 handler: async ({ name, username, password }) => {
 
                     const client = getClient();
-                    const { data: results } = await client.post('/register', {
+                    await client.post('/register', {
                         name,
                         username,
                         password
                     }, {
                         reauthorize: false
                     });
 
-                    if (results.data.accessToken) {
-                        Storage.setItem('accessToken', results.data.accessToken);
-                    }
-
-                    await m.dispatch.auth.fetchCurrentUser();
+                    await m.dispatch.auth.login({ username, password });
```

Comment:
UX-wise, I'm not sure if we want to automatically log someone in after registering, but found pallies does not return tokens from the register route, so at the very least, we'd remove storing the token.

Reply:
I like the flow of logging someone in after registering them, just so that after inputting a username and password, the user isn't just redirected to a new page and asked to enter their username and password. I'm open to other perspectives, though!